syntax = "proto3";

package distflow.protos;

// first byte contains type, followed by header, array data saved in following code.

message Header {
  fixed32 service_id = 1;
  fixed32 chunk_index = 2;
  fixed64 rpc_id = 3;
  string data = 4;
  string service_key = 5;
}

